services/backend/services/v3/compose.base.yaml

```diff
@@ -8,7 +8,13 @@ services:
     volumes:
       - ./config/config.local.js:/home/node/app/server/config.local.js
       - ./config/datasources.json:/home/node/app/server/datasources.json
-      - ./config/providers.json:/home/node/app/server/providers.json
+      - ./config/providers.json:/config/providers.base.json
+      - ./config/init.sh:/init.sh
+      - ./config/mergeJson.js:/config/mergeJson.js
+    command:
```

Comment:
Doesn't this node image have the docker-entrypoint folder?

Reply:
No, it only has `docker-entrypoint.sh` 
```
docker run --rm -it ghcr.io/scicatproject/backend-next:v4.5.0 /bin/cat /usr/local/bin/docker-entrypoint.sh
#!/bin/sh
set -e

# Run command with node if the first argument contains a "-" or is not a system command. The last
# part inside the "{}" is a workaround for the following bug in ash/dash:
# https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=874264
if [ "${1#-}" != "${1}" ] || [ -z "$(command -v "${1}")" ] || { [ -f "${1}" ] && ! [ -x "${1}" ]; }; then
  set -- node "$@"
fi

exec "$@"
```